tool/gravity/cli/clusterupdate.go

```diff
@@ -217,6 +217,14 @@ func executeUpdatePhase(env *localenv.LocalEnvironment, environ LocalEnvironment
 }
 
 func executeUpdatePhaseForOperation(env *localenv.LocalEnvironment, environ LocalEnvironmentFactory, params PhaseParams, operation ops.SiteOperation) error {
+	allActive, err := verifyActiveAgents(env)
+	if err != nil {
+		return trace.Wrap(err, "failed to verify agent status")
+	}
+	if !allActive {
+		return trace.BadParameter("some agents are offline; ensure all agents are deployed with `gravity agent deploy`")
```

Comment:
Any way to just show the same table as "agent status" command shows to the user doesn't have to guess which are offline?